.github/workflows/tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Build
+
+on: [push, pull_request]
```

Comment:
This seems to cause actions to be duplicated; I think it's a known issue (https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012) there are a few workarounds in that thread but they don't seem great

Reply:
I am OK with duplicate actions as the least dangerous/ugly thing: otherwise a direct push to `master` without PR could be problematic. A maybe cleaner approach is to have only `master` checked on `push`, and anything else on `pull_request`. What do you think?

Reply:
better to have too many than miss one; if you can do the master-on-push everything-else-on-pr that's probably best, but otherwise this is better than missing commits yes